src/ClickStrategyApplicator/CompositeClickStrategyApplicator.php

```diff
@@ -11,11 +11,11 @@ final class CompositeClickStrategyApplicator extends ArrayCollection implements
     /** @var string */
     private $clickStrategy;
 
-    public function __construct(string $clickStrategy)
+    public function __construct(array $elements = [], string $clickStrategy = '')
```

Comment:
This is a BC break and should not be in this PR

Reply:
This is now reverted (with suppressed psalm errors)